src/tests/verified_chats.rs

```diff
@@ -782,6 +782,33 @@ async fn test_create_oneonone_chat_with_former_verified_contact() -> Result<()>
     Ok(())
 }
 
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn test_create_verified_multidev() -> Result<()> {
+    let mut tcm = TestContextManager::new();
+    let alice = tcm.alice().await;
+    let alice1 = tcm.alice().await;
+
+    let group_id = alice
+        .create_group_with_members(ProtectionStatus::Protected, "Group", &[])
+        .await;
+    assert_eq!(
+        get_chat_msg(&alice, group_id, 0, 1).await.get_info_type(),
+        SystemMessage::ChatProtectionEnabled
+    );
+
+    let sent = alice.send_text(group_id, "Hey").await;
+    tokio::time::sleep(std::time::Duration::from_millis(2000)).await;
```

Comment:
is it too cumbersome to wait for appropriate events instead of a magic sleep?

Reply:
Added a comment about why we need this sleep. So, the test passes even w/o it, but then the bug isn't reproduced for the old code.

Reply:
Isn't it enough to wait 1100ms, just as test_handle_authres_fails(), test_modify_chat_disordered(), and test_modify_chat_lost()?

Reply:
It's not enough because of the logic inside `smeared_time()`. It should be even 2100 ms then, but 2000 is enough to reproduce the bug for the old code with almost 100% probability. In other tests these extra 100 ms are to avoid spurious failures which are not the case here.